drivers/serial/uart_pl011.c

```diff
@@ -286,14 +290,16 @@ static const struct uart_driver_api pl011_driver_api = {
 #endif /* CONFIG_UART_INTERRUPT_DRIVEN */
 };
 
-static int pl011_init(const struct device *dev)
+static int _pl011_init(const struct device *dev, bool early)
 {
 	const struct pl011_config *config = dev->config;
 	struct pl011_data *data = dev->data;
 	int ret;
 	uint32_t lcrh;
 
+#ifndef CONFIG_ARM_VERY_EARLY_CONSOLE
 	DEVICE_MMIO_MAP(dev, K_MEM_CACHE_NONE);
+#endif
```

Comment:
When multiple pl011 devices are enabled in the devicetree, others are not mapped except the one the console selects.

Reply:
Good point, I need to fix it.